.circleci/wait-for-job-finish.sh

```diff
@@ -12,7 +12,7 @@ counter=0
 while [ "$counter" -le 1500 ]; do
   # Find number of nodes in running
   job_detail=$(curl -s "https://circleci.com/api/v2/project/github/DataBiosphere/terra-ui/job/$CIRCLE_BUILD_NUM")
-  job_running_nodes_count=$(echo "$job_detail" | jq -r '.parallel_runs[] | select(.status == "running") | select(.index != '"$CIRCLE_NODE_INDEX"')' | grep -c "running")
+  job_running_nodes_count=$(echo "$job_detail" | jq -r '[.parallel_runs[] | select(.status == "running") | select(.index != '"$CIRCLE_NODE_INDEX"')] | length')
```

Comment:
I think my draft [PR](https://github.com/DataBiosphere/terra-ui/pull/4833/files) would have solved this problem too in a different way, by `|| test $? = 1;`.  It's just I did not have a chance to finish it up.